frontend/packages/process-editor/src/components/Canvas/BPMNViewer/BPMNViewer.tsx

```diff
@@ -1,33 +1,22 @@
 import React, { ReactNode } from 'react';
 import classes from './BPMNViewer.module.css';
 import { useBpmnViewer } from '../../../hooks/useBpmnViewer';
-import { VersionAlert } from './VersionAlert';
 import { BPMNViewerErrorAlert } from './BPMNViewerErrorAlert';
-import { supportsProcessEditor } from '../../../utils/processEditorUtils';
-
-export type BPMNViewerProps = {
-  appLibVersion: string;
-};
 
 /**
  * @component
  *  Displays the canvas area of the ProcessEditor
  *
- * @property {string}[appLibVersion] - The app-lib version the user has
- *
  * @returns {ReactNode} - The rendered component
  */
-export const BPMNViewer = ({ appLibVersion }: BPMNViewerProps): ReactNode => {
+export const BPMNViewer = (): ReactNode => {
```

Comment:
I know this is a little out of scope for the PR, but could we change the return type to `JSX.Element` instead of `ReactNode`? 🙈 

ReactNode can be a lot of things like `null | undefined | boolean` etc, we are not sure that the return is a `JSX.Element `which we expects here. 😊 If you decide to change this, the JSDoc must be updated as well.
```suggestion
export const BPMNViewer = (): JSX.Element => {
```